services/web/server/src/simcore_service_webserver/director_v2.py

```diff
@@ -176,6 +175,23 @@ async def stop_pipeline(request: web.Request) -> web.Response:
         )
 
 
+@log_decorator(logger=log)
+async def request_retrieve_dyn_service(
+    app: web.Application, service_uuid: str, port_keys: List[str]
+) -> None:
+    director2_settings: Directorv2Settings = get_settings(app)
+    backend_url = URL(
+        f"{director2_settings.endpoint}/dynamic_services/{service_uuid}:retrieve"
+    )
+    body = {"port_keys": port_keys}
+
+    try:
+        # request to director-v2
+        await _request_director_v2(app, "POST", backend_url, data=body)
+    except _DirectorServiceError:
+        pass
```

Comment:
is this desired?

Reply:
good one, I added a warning now to log in case of issues.